client/tutorials/ibm-blockchain-platform-vscode-smart-contract/local-dev.md

```diff
@@ -156,7 +201,9 @@ To do this...
 
 1. In the `LOCAL FABRIC OPS` panel, look for `+ Install` (it's under Smart Contracts > Installed) and click it.
 
-2. You'll be asked to choose a package to install. Pick `demoContract@0.0.1`.
+2. You'll be asked to choose a peer. Pick the only option: `peer0.org1.example.com`.
```

Comment:
does it really ask you to choose a peer if there is only 1? if it does it shouldn't